src/meta/src/stream/stream_graph.rs

```diff
@@ -1118,6 +1134,9 @@ impl ActorGraphBuilder {
             NodeBody::TopN(node) => {
                 vec![node.table.as_ref().unwrap().id]
             }
+            NodeBody::Sink(node) => {
```

Comment:
Since `Sink` doesn't have any state tables in hummock, maybe it's not necessary to record it in state table set in fragment. Or we'd better use `SinkCatalog` instead? 🤔

Reply:
I'll remove it